hail/src/main/scala/is/hail/backend/service/ServiceBackend.scala

```diff
@@ -76,14 +66,15 @@ object Worker {
 class ServiceBackendContext(
   val username: String,
   @transient val sessionID: String,
-  val billingProject: String
+  val billingProject: String,
+  val bucket: String
 ) extends BackendContext
 
 object ServiceBackend {
-  lazy val log = LogManager.getLogger("ServiceBackend")
+  lazy val log = LogManager.getLogger("is.hail.backend.service.ServiceBackend")
```

Comment:
why this change?

Reply:
Consistency.  I used the fully qualified class name elsewhere.  I don't have a huge preference, although there could be name collisions otherwise.